benchto-driver/src/main/java/io/trino/benchto/driver/BenchmarkProperties.java

```diff
@@ -233,4 +241,9 @@ else if (booleanString.equalsIgnoreCase(Boolean.FALSE.toString())) {
             throw new IllegalStateException(String.format("Incorrect boolean value: %s.", booleanString));
         }
     }
+
+    public enum RepeatLevel {
+        QUERY,
+        BENCHMARK
```

Comment:
* if repeat level is benchmark with count = 5 does it mean I have 5 separate benchmark results with single query execution in each benchmark?
* if repeat level is query with count = 5 does it mean I have 1 benchmark result with query repeated 5 times?

Maybe it should be called `BENCHMARK` -> `SUITE` (aka repeat queries globally withing entire suite execution)

Reply:
> if repeat level is benchmark with count = 5 does it mean I have 5 separate benchmark results with single query execution in each benchmark?

No it does not. Unfortunately, there is a lot of confusion in the term `benchmark`. Inside the benchto driver the `benchmark` is the instance of one combination that can be generated from `yaml` file.
For `BENCHMARK_LEVEL` the `benchto-driver` repeats execution of list of benchmarks (list of different combination generated from `yaml`). 
For instance: 
If we have two benchmarks: `b1, b2` and benchmarks have "attached" queries, `(q1, q2)` and `(q3, q4)` respectively and level is `BENCHMARK` and `runs = 2` then:
`(q1, q2, q3, q4), (q1, q2, q3, q4)`

There will be one *seperate* result. 

> Maybe it should be called BENCHMARK -> SUITE (aka repeat queries globally withing entire suite execution)

Good point